Experience.html

```diff
@@ -98,19 +109,13 @@ <h2 class="UNICEF">Physical Education Teacher (UNICEF, Syria)
                         <li>Coaching social sporting teams
                         </li>
                     </ul>
-        
-
-
-       
-        
-
-
-      
-   
-    
-    
-    
-       
+                  </nav>
+         
+                  <footer>
+                     <p class="copyright"> copyright 2020 reem alawad
+                    </p>
+                   </footer>  
 
     </main>
+   </body>
 </html>
```

Comment:
Thinking how about we let a HTML formatter called `beautifier` fix the indentation for us? Here's how:

1. Go to https://beautifier.io/
2. Set the configuration options as per the screenshow
3. Paste one of your HTML files into the tool
4. Click Beautify Code
5. Paste result back into your editor
6. Continue steps 3-5 for all other HTML files in your project
7. Commit all changes and push up

![Screenshot from 2020-11-25 11-26-05](https://user-images.githubusercontent.com/394074/100166687-66c3f580-2f11-11eb-8870-afc58347eef2.png)